stable/airflow/templates/configmap-env.yaml

```diff
@@ -14,28 +14,36 @@ data:
   POSTGRES_HOST: "{{ template "airflow.postgresql.fullname" . }}"
   POSTGRES_PORT: "{{ .Values.postgresql.service.port }}"
   POSTGRES_DB: "{{ .Values.postgresql.postgresDatabase }}"
+  {{- if .Values.redis.enabled }}
```

Comment:
'redis.enabled=true' actually creates a redis instance, while user can still provide the url to their own one with redis.enabled=false.

You can move under the `if eq .Values.airflow.executor "Celery"` unbrella

Reply:
done